docs/source/ecf.rst

```diff
@@ -261,6 +261,9 @@ skip_*
 ''''''
 If True, skip the named step.
 
+.. note::
+   To produce flux-calibrated stellar spectra, it is recommended to set ``skip_flat_field`` and ``skip_photom`` to ``False``, and potentially to also set ``skip_extract_1d`` to ``False`` as well.
```

Comment:
> and potentially to also set ``skip_extract_1d`` to ``False`` as well.

Where does this recommendation come from?

Reply:
Was just in case people want a quick-look spectrum, but I'll remove it